inline-imports.js

```diff
@@ -0,0 +1,47 @@
+/** @format */
+function transformIt( babel ) {
```

Comment:
1. would it be easy to add a few tests for this?
2. do you think its important to use actual action-type values instead of the import name?

Reply:
1. No - testing this code I think would be quite hard because of what we're trying to validate.
2. No - the only important thing is that they are unique, which the import semantics already constrain us on, so if it were invalid it wouldn't make it this far (assuming the CommonJS transform is off)

In fact we should be more resilient to typos here after inlining because if someone accidentally copies an action type value on another key we will filter out collisions. I already discovered something like this in reader code where we were passing an undefined constant - that only happened to work in practice because of the constants that were being used none of the others happened to have the "value" of `undefined`, but had we had typos on two separate constant names we would have collided.